Common/Transforms/elxTransformIO.cxx

```diff
@@ -79,6 +106,24 @@ elastix::TransformIO::Write(const itk::TransformBase & itkTransform, const std::
 }
 
 
+itk::SmartPointer<itk::TransformBase>
```

Comment:
If I remember correctly ITK has the habit to return normal pointers, which are then convertible to smartpointer. There is probably a reason behind that habit, please check.

Reply:
Thanks for looking at this, Marius. In this case I think returning a smart pointer is fine, to reduce the chance that the caller will accidentally use a normal pointer.

You see, the following would crash:

    Object* GetNormalPointer(void)
    {
        return Object::New();
    }

    int main(void)
    {
        Object* object = GetNormalPointer();
        object->Print(std::cout);  // Crash! Object is already destructed!
    }